analyzer/consensus/consensus.go

```diff
@@ -921,6 +921,33 @@ func (m *processor) queueEscrows(batch *storage.QueryBatch, data *stakingData) e
 			e.Owner.String(),
 			e.ActiveShares.String(),
 		)
+		// Ideally this would be merged with the ConsensusDebondingStartDelegationsUpdate query above,
```

Comment:
@mitjat note that the previous version didn't actually work, i had to refactor the CTE into two separate queries.